### PR TITLE
[MIG] website_addthis: migration to v12

### PR DESCRIPTION
**Pull request by JonathanOsAlc**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please preserve commit history following technical method explained in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0

File: website_addthis/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2018 Camptocamp
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "AddThis integration",
+    "version": "12.0.0.0.0",
```
Comment (tbaden) [inline website_addthis/__manifest__.py:6]: Version should be: 12.0.1.0.0

File: website_addthis/views/website.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_website_form_addthis" model="ir.ui.view">
```
Comment (tbaden) [inline website_addthis/views/website.xml:4]: aesthetical change (optional):
use original id: view_website_form

> ... prefer naming the inherited view after the original over using a name which follows the guidelines. This eases looking up the original view and other inheritance if they all have the same name.

File: website_addthis/views/website_config_settings.xml [hunk 3]
```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="res_config_settings_view_form_website" model="ir.ui.view">
```
Comment (tbaden) [inline website_addthis/views/website_config_settings.xml:4]: aesthetical change (optional):
use original id: res_config_settings_view_form

> ... prefer naming the inherited view after the original over using a name which follows the guidelines. This eases looking up the original view and other inheritance if they all have the same name.

**Review by tbaden — CHANGES_REQUESTED** [review]
thanks for the work 👍 
i did a code review. here some minor change + two optional changes:

File: website_addthis/views/website_config_settings.xml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="website.res_config_settings_view_form" />
+        <field name="arch" type="xml">
+            <div id="website_settings" position="inside">
+                <div class="col-xs-12 col-md-6 o_setting_box" id="addthis">
```
Comment (sergio-teruel) [inline website_addthis/views/website_config_settings.xml:9]: Change col-xs-12 to col-12
see here: https://getbootstrap.com/docs/4.0/migration/

File: website_addthis/views/website_config_settings.xml [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+                        <field name="addthis_enabled"/>
+                    </div>
+                    <div class="o_setting_right_pane">
+                        <label for="addthis_enabled"/>
+                        <div class="text-muted">
+                            Configure AddThis integration
+                        </div>
+                        <div class="content-group" attrs="{'invisible': [('addthis_enabled', '=', False)]}">
+                            <div class="row mt16">
+                                <label class="col-md-3 o_light_label" for="addthis_pubid"/>
```
Comment (sergio-teruel) [inline website_addthis/views/website_config_settings.xml:20]: col-md is col-lg in bootstrap v4

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

File: website_addthis/views/website.xml [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+        <field name="model">website</field>
+        <field name="inherit_id" ref="website.view_website_form"/>
+        <field name="arch" type="xml">
+            <div name="other" position="after">
+                <group name="Addthis Integration">
```
Comment (simahawk) [inline website_addthis/views/website.xml:9]: `addthis_integration`. If you want you can keep your naming but as `string`.

**Review by simahawk — CHANGES_REQUESTED** [review]

**Comment by simahawk** [comment]
@JonathanOsAlc ping :)

**Comment by Tardo** [comment]
@JonathanOsAlc ping (:

**Pull request closed without merging**